src/components/User/Limits/components/Limits.Manager.vue

```diff
@@ -317,77 +431,128 @@ export default {
+        await ApiCallerFactory
+          .createCallerInstance()
+          .postOperations(operation)
         await this.getAssets()
         this.$store.dispatch('SET_INFO', 'Limits update saved')
       } catch (e) {
         ErrorHandler.process(e)
       }
       this.isPending = false
     },
-    async getAssets () {
+    async removeLimits (limits, type) {
+      if (limits.id === 0) {
+        return false
+      }
+      /* eslint-disable-next-line max-len */
+      if (!await confirmAction({ title: this.getTitleConfirmAction(type) })) return
+      this.isPending = true
       try {
-        const response = await Sdk.horizon.assets.getAll()
-        this.assets = response.data
+        const operation = Sdk.base.ManageLimitsBuilder.removeLimits({
+          id: (limits.id).toString(),
+        })
+        await ApiCallerFactory
+          .createCallerInstance()
+          .postOperations(operation)
+        await this.getLimits()
+        this.$store.dispatch('SET_INFO', 'Limits removed')
       } catch (e) {
-        ErrorHandler.processWithoutFeedback(e)
+        ErrorHandler.process(e)
       }
+      this.isPending = false
     },
-    async getRequestorAccountId (requestor) {
-      if (Sdk.base.Keypair.isValidPublicKey(requestor)) {
-        return requestor
+    getTitleConfirmAction (type) {
+      if (this.filters.scope === SCOPE_TYPES.accountRole) {
+        const accountRole = this.ACCOUNT_ROLES_VERBOSE[this.filters.accountRole]
+          .toLowerCase()
+
+        return `Delete ${accountRole} ${type} limits?`
       } else {
-        try {
-          const address = await api.users.getAccountIdByEmail(requestor)
-          return address || requestor
-        } catch (error) {
-          return requestor
+        return `Delete ${this.userEmail} ${type} limits?`
+      }
+    },
+    async getAssets () {
+      const { data } = await ApiCallerFactory
+        .createStubbornCallerInstance()
+        .stubbornGet('/v3/assets')
+      this.assets = data
+    },
+    async loadAccountIdByEmail (email) {
+      try {
+        const { data } = await ApiCallerFactory
+          .createCallerInstance()
+          .getWithSignature('/identities', {
+            filter: { email: email },
+            page: { limit: 1 },
+          })
+        this.userEmail = ((data || [])[0] || {}).email || email
+        if (this.userEmail === email) {
+          this.filters.address = ((data || [])[0] || {}).address
+        } else {
+          this.filters.address = ''
         }
+      } catch (error) {
+        ErrorHandler.processWithoutFeedback(error)
       }
     },
     // it's a quick fix of the limits validation. Need to refactor it ASAP
     isValidLimits (limits) {
       for (const limit of Object.values(pick(limits, LIMITS_TYPES))) {
-        if (!this.numericValueRegExp.test(limit)) {
+        if (limit.value === null) {
+          ErrorHandler.process('Fill in all the fields. Not set value not allowed')
```

Comment:
really? I remember I could set only daily limits, for example. You cannot set weekly < daily, for example, but you can omit setting limits of the larger period. correct me if I’m wrong pls